Description: foomatic-rip: SECURITY FIX: Also consider the back tick ('`') as
 an illegal shell escape character. Thanks to Michal Kowalczyk from the Google
 Security Team for the hint.
Author: Till Kamppeter <till.kamppeter@gmail.com>
Bug-CVE: CVE-2015-8327
Origin: upstream
Last-Update: 2015-11-26

--- a/filter/foomatic-rip/util.c
+++ b/filter/foomatic-rip/util.c
@@ -31,7 +31,7 @@
 #include <assert.h>
 
 
-const char* shellescapes = "|<>&!$\'\"#*?()[]{}";
+const char* shellescapes = "|<>&!$\'\"`#*?()[]{}";
 
 const char * temp_dir()
 {
